src/openhmd.c

```diff
@@ -97,6 +98,18 @@ int OHMD_APIENTRY ohmd_ctx_probe(ohmd_context* ctx)
 	return ctx->list.num_devices;
 }
 
+const char* OHMD_APIENTRY ohmd_gets(ohmd_string_description type)
```

Comment:
Yes, the gets functions are for statically allocated strings. We need to add a new one for dynamically allocated strings, if that's required. No biggie though. ohmd_gets_rw()?